src/ripple/test/jtx/JTx.h

```diff
@@ -56,12 +59,99 @@ struct JTx
     {
     }
 
+#if defined(_MSC_VER) && _MSC_VER <= 1800
+    JTx(JTx&& src)
```

Comment:
The confusion was my fault for putting the constructors in the wrong order. I've reordered them.